helper/benchmark_output.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+    Benchmark Output Structure
+    This class serves as a data structure to contain the parsed (or not)
+    results of the benchmark, the dictionary of parsed perf results of running
+    the benchmark, and the metadata associated to this run of the benchmark
+
+"""
+
+
+class BenchmarkOutput(object):
+    def __init__(self):
+        self.stdout_list = []
+        self.perf_results = []
```

Comment:
don't call this perf_results, it's really stderr. It's up to the caller to know what stderr stands for.